Gutenberg/cocoapods_helpers.rb

```diff
@@ -207,3 +213,22 @@ def workaround_broken_search_paths
   end
   project.save
 end
+
+# Copy Gutenberg bundle and source map files so they can be upload it to Sentry during the build process.
+def extract_bundle_source_map_files
+  puts '[Gutenberg] Extracting bundle and source map files'
+
+  FileUtils.mkdir_p(GUTENBERG_BUNDLE_SOURCE_MAP_TARGET)
+  bundle_from_path = File.join(GUTENBERG_FRAMEWORK_FOLDER, 'App.js')
+  bundle_destination_path = File.join(GUTENBERG_BUNDLE_SOURCE_MAP_TARGET, 'main.jsbundle')
+  FileUtils.cp(bundle_from_path, bundle_destination_path)
+
+  # Source map file is moved instead of copied to avoid including it in the binary.
+  source_map_from_path = File.join(GUTENBERG_FRAMEWORK_FOLDER, 'App.composed.js.map')
+  source_map_destination_path = File.join(GUTENBERG_BUNDLE_SOURCE_MAP_TARGET, 'main.jsbundle.map')
+  if File.exist?(source_map_from_path)
+    FileUtils.mv(source_map_from_path, source_map_destination_path)
```

Comment:
I haven't checked yet but I suspect that caching Pods ([reference](https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin/blob/f077728b25454cd7e727481096306603ae33d782/bin/install_cocoapods#L61)) might conflict with this approach of moving the source map file outside the `Pods` folder. If the cache is generated after the file is moved, the next run might fail because the file won't exist.

As a workaround, we could create the `react-native-bundle-source-map` folder under `Pods/Gutenberg`. This would ensure that the files are part of the Pods cache. WDYT?

Reply:
Yeah I think it'd be best to have it under the Pods folder to avoid that potential issue

Reply:
I've just pushed an updated to avoid this potential issue: https://github.com/wordpress-mobile/WordPress-iOS/pull/22862/commits/8dbe78e59843c4ea061f907e9261eb7bfe88a7a2.

I'm also working on addressing this in GBM instead of here (https://github.com/wordpress-mobile/gutenberg-mobile/pull/6760). However, I'd rather pursue merging this PR to include it in beta version 24.5 and avoid cutting a GMB beta fix.

Reply:
>I'm also working on addressing this in GBM instead of here (https://github.com/wordpress-mobile/gutenberg-mobile/pull/6760). However, I'd rather pursue merging this PR to include it in beta version 24.5 and avoid cutting a GMB beta fix.

Awesome! Thank you for that!

Reply:
@geriux Here's the PR with a more optimal approach, let me know if you could take a look (no rush!). Thanks 🙇  !